src/DSA/BFSGraphMatrix/BFSGraphMatrix_test.go

```diff
@@ -0,0 +1,20 @@
+package bfsgraphmatrix
+
+import (
+	"testing"
+
+	dsa "github.com/nacknime-official/kata-machine-go/src/DSA"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestBfs(t *testing.T) {
+	matrix := dsa.AdjMatrix1
+
+	result := Bfs(matrix, 6, 0)
+	expected := []int{0, 1, 4, 5, 6}
+	assert.Equal(t, expected, result, "Array should be: %v\nBut got: %v", expected, result)
+
+	result = Bfs(matrix, 0, 6)
```

Comment:
Seems you swapped accidentally the arguments
https://github.com/ThePrimeagen/kata-machine/blob/d9d8d06b8a1a2c9058168eb8ae059aec80d4e24c/src/__tests__/BFSGraphMatrix.ts#L13